paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/SupportedPaymentMethod.kt

```diff
@@ -0,0 +1,50 @@
+package com.stripe.android.paymentsheet.model
+
+import android.os.Parcelable
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import com.stripe.android.paymentsheet.R
+import com.stripe.android.paymentsheet.specifications.bancontact
+import com.stripe.android.paymentsheet.specifications.sofort
+import kotlinx.parcelize.Parcelize
+
+/**
+ * Enum defining all payment methods supported in Payment Sheet.
+ */
+@Parcelize
```

Comment:
I believe `enum` doesn't need this.

Reply:
It does, otherwise I get "This class implements Parcelable but does not provide a CREATOR field"

Reply:
Nevermind, I can make it non-Parcelable and use the name.

Reply:
You don't need to implement `Parcelable`